src/routes/external.route.ts

```diff
@@ -33,6 +33,12 @@ class ExternalRoute implements Routes {
   constructor() {
     this.initializeAuthRoutes(`${this.path}/auth`);
     this.initializeSchoolRoutes(`${this.path}/school`);
```

Comment:
remove or mark all school related api as deprecated.
also update script to call internally to pathshala service for creation of school.